src/main/java/org/jf/fusionIdea/run/FusionScriptState.java

```diff
@@ -236,40 +242,15 @@ private MulticastSocket sendIpv4SSDPRequest() {
 
         private MulticastSocket sendIpv6SSDPRequest() {
             try {
-                Enumeration<NetworkInterface> nets = NetworkInterface.getNetworkInterfaces();
-
-                InetSocketAddress multicastAddress = new InetSocketAddress(
-                        "ff01:fb68:e6b7:45f9:4acc:2559:6c6e:c014", 1900);
-                MulticastSocket socket = new MulticastSocket(0);
-                socket.setLoopbackMode(/* disabled= */ false);
-
-                boolean success = false;
-                for (NetworkInterface netint : Collections.list(nets)) {
-                    try {
-                        if (netint.supportsMulticast()) {
-                            boolean hasIpV6 = false;
-                            for (InetAddress address : Collections.list(netint.getInetAddresses())) {
-                                if (address instanceof Inet6Address) {
-                                    hasIpV6 = true;
-                                    break;
-                                }
-                            }
-                            if (hasIpV6) {
-                                socket.setNetworkInterface(netint);
-                                socket.send(new DatagramPacket(SEARCH_MESSAGE, SEARCH_MESSAGE.length, multicastAddress));
-                                success = true;
-                            }
-                        }
-                    } catch (IOException ex) {
-                        FusionIdeaPlugin.log.debug("ipv6 multicast failed on " + netint.getName(), ex);
-                    }
-                }
-
-                if (!success) {
-                    FusionIdeaPlugin.log.warn("Couldn't send ipv6 ssdp packet on any interface");
-                    return null;
-                }
-
+                InetAddress LOCALHOST_IPV6 = Inet6Address.getByName("::1");
+                MulticastSocket socket = new MulticastSocket(new InetSocketAddress(LOCALHOST_IPV6, 0));
```

Comment:
This works on windows if I don't specify an interface for this at all. e.g. 

```java
                MulticastSocket socket = new MulticastSocket();
                socket.setLoopbackMode(false);
                DatagramPacket packet = new DatagramPacket(
                    SEARCH_MESSAGE, SEARCH_MESSAGE.length,
                    new InetSocketAddress(
                        Inet6Address.getByName(MULTICAST_GROUP_IPV6), MULTICAST_PORT));
```

Reply:
See also my comments in the other PR.

If the above change doesn't work for mac, we'll probably need to do a platform check here too.

Reply:
I added the platform check. Somehow on the Mac it appears to be necessary to both set the socket on localhost and to explicitly set the interface.